rpcadaptors.go

```diff
@@ -464,3 +465,39 @@ var _ rpcserver.FeeEstimator = (*rpcFeeEstimator)(nil)
 type rpcFeeEstimator struct {
 	*fees.Estimator
 }
+
+// Ensure rpcBlockTemplater implements the rpcserver.Templater interface.
+var _ rpcserver.BlockTemplater = (*rpcBlockTemplater)(nil)
+
+// rpcBlockTemplater provides a block template generator for use with the
+// RPC server and implements the rpcserver.BlockTemplater interface.
+type rpcBlockTemplater struct {
+	g *mining.BgBlkTmplGenerator
+}
+
+// ForceRegen asks the block templater to generate a new template immediately.
+func (b *rpcBlockTemplater) ForceRegen() {
```

Comment:
If the generator is embedded as suggested above, this, and the rest should be able to go away.